docs/DeveloperGuide.md

```diff
@@ -423,6 +372,35 @@ use this index when deleting tasks
   would be troublesome and could lead to bugs. In alternative 1, all indexes are flushed to the front (i.e first task
   has index 1, second task has index 2 etc) and thus the invalid indexes can be easily obtained.
 
+## Unassigning a Task from a Person
```

Comment:
I think it would be better if you could sync the fork to include the latest @Guo-KeCheng 's numbering (as he has labeled each section of the DG with numbers now). Resolving the merge conflicts here on the spot may not be a good idea as it may overwrite the git blame of the code (eg. if you modify nicholas's code to include the numbering, the git blame may be pointing to you for his DG stuff instead)